app/js/components/actions/severitySummary/severitySummary.jade

```diff
@@ -3,8 +3,8 @@
     li
       .pointer(ng-if="ruleCount.info || !loaded", ui-sref="app.topic({id: 'low-risk', 'filters:totalRisk': 'INFO'})")
         .metrics
-          strong {{'Low' | translate}}
-          span.num(ng-if="(ruleCount.info / ruleCount.total) < 80") &nbsp;({{100 * ruleCount.info / ruleCount.total || 0 | number:1}}%)
+          strong {{:: 'Low' | translate}}
+          span.num(ng-if="(ruleCount.info / ruleCount.total) < 80") &nbsp;({{:: 100 * ruleCount.info / ruleCount.total || 0 | number:1}}%)
```

Comment:
This breaks on the overview page when changing groups. Notice the counts:

![screenshot_20180117_103300](https://user-images.githubusercontent.com/167352/35035491-d6298c18-fb71-11e7-9b5a-203370f7baac.png)

Reply:
Should be fixed... hoisted the ng-if up to let the loading catch/rerender these things.

Reply:
this is fixed now

Reply:
actually no, it's still broken. The bar width is updates as expected but the percents aren't